### ISSUE TITLE
Improve text for DFG template question 1.2

### ISSUE THREAD

**Issue by StellaEggels** [nfdi4plants/dataplan#49]
I think the sentence below will often not fit well following the project aim if the project does not focus on data analysis. And the second part of the sentence " but also be informed..." does not make sense to me. I would replace that part of the sentence by something else, because in the sentence after provenance is also mentioned.

Therefore, data collection, integration and visualization through a standardized data management process is absolutely necessary, because the data are used not only to understand principles, but also be informed about the provenance of data analysis information.

**Comment by StellaEggels** [comment]
the same is written under 2.2

**Comment by StellaEggels** [comment]
And the first sentence of 1.2 does not say much: 
Data of different types or of different domains will be generated differently

**Comment by xiaoranzhou** [comment]
> And the first sentence of 1.2 does not say much: Data of different types or of different domains will be generated differently

I will rephrase it to: Data collection or generation methods may vary depending on their types and domains

**Comment by xiaoranzhou** [comment]
> the same is written under 2.2

I think adding this sentense might help:
Using methods from question 1.2 ensures high-quality raw data during collection and generation
